terraform/executor/Dockerfile-0.12-slim

```diff
@@ -12,7 +12,7 @@ ARG AWSCLI_VERSION=1.18.134
 ARG HELM2_VERSION=2.16.10
 ARG HELM3_VERSION=3.3.1
 ARG TFSWITCH_VERSION=0.8.832
-ARG TF_AIVEN_VERSIONS="2.0.1"
+ARG TF_AIVEN_VERSIONS="2.0.1 2.0.6"
```

Comment:
latest v2 because they need 0.13 to download via tf, and 1 was never made available in slim